### DISCUSSION TITLE
Running the human from ubuntu

### DISCUSSION THREAD

**Discussion by saeidn95** [vladmandic/human#439] [Q&A]
I am not familiar with js. To follow through the code, I tried run human by executing "node human.js" from the "human/src" folder on a terminal on ubuntu, and also from the "human/dist" it did not work. I would be good to have some instruction how to run it from ubuntu platform.

**Comment by vladmandic** [comment]
this is not about ubuntu at all, you don't run a library directly.
you can run any demo that is included - and they are listed in the readme.

**Reply by saeidn95** [reply]
I can run the demos online with a webcam. I like to trace through the code, insert some print statements and debug to understand it better.

**Reply by vladmandic** [reply]
if you want to modify library code, you need to rebuilt it and then run demo or your app.
and how to rebuild library is documented.
